federation/pkg/federatedtypes/crudtester/crudtester.go

```diff
@@ -29,6 +29,7 @@ import (
 
 const (
 	AnnotationTestFederationCRUDUpdate string = "federation.kubernetes.io/test-federation-crud-update"
+	ObjectNotExpectedWaitTime                 = 10 * time.Second
```

Comment:
That's fair. I'll remove the pieces that try to reduce test time.  We could switch to an order of magnitude of the last waitFroResource time but it's probably not worth the effort unless 30s for this test to run is a concern.